mm/2s2h/GameInteractor/GameInteractor.h

```diff
@@ -217,6 +217,7 @@ typedef enum {
     VB_GIBDO_TRADE_SEQUENCE_DO_TRADE,
     VB_GET_ITEM_ACTION_FROM_MASK,
     VB_GRANT_MAGIC_UPON_REQUEST,
+    VB_SCOPENUTS_HAVE_NO_OCARINA,
```

Comment:
Maybe? the way it is now makes it sound like the scrub has an ocarina lol
```suggestion
    VB_SCOPENUTS_CONSIDER_FIRST_CYCLE,
```